erc20-connector/test/TokenLocker.js

```diff
@@ -100,12 +210,12 @@ contract('TokenLocker', function ([addr, addr1]) {
     });*/
 
     it('admin functions', async function() {
-      expect(await this.locker.admin_()).equal(addr)
-      try {
-        await this.locker.adminTransfer(this.token.address, addr1, toWei('1'), { from: addr1 })
-        assert(false)
-      } catch (_) { }
-      await this.locker.adminTransfer(this.token.address, addr1, toWei('1'), { from: addr })
-      expect(fromWei(await this.token.balanceOf(addr1))).to.be.equal('6')
+        expect(await this.locker.admin()).equal(addr)
+        try {
```

Comment:
Check indents

Reply:
These are ok and correspond to the file's indents.